frontend/catalyst/compilation_pipelines.py

```diff
@@ -52,7 +52,6 @@
 
 jax.config.update("jax_enable_x64", True)
 jax.config.update("jax_platform_name", "cpu")
-jax.config.update("jax_array", True)
```

Comment:
Isn't this required anymore for enabling `jax.Array`? Is this deprecated or we never really used this feature?